facebook/__init__.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, access_token=None, timeout=None, version=None,
             # https://developers.facebook.com/docs/graph-api/securing-requests
             self.app_secret_hmac = hmac.new(app_secret.encode('ascii'),
                                             msg=access_token.encode('ascii'),
-                                            digestmod=hashlib.sha256)
+                                            digestmod=hashlib.sha256).hexdigest()
```

Comment:
It would be great if this commit's message explained why we are using hexdigest (e.g. "Use hexdigest instead of a bare HMAC object because Facebook expects a hexadecimal hash of the HMAC.").